modules/apps/27-interchain-accounts/host/ibc_module.go

```diff
@@ -105,14 +107,19 @@ func (im IBCModule) OnRecvPacket(
 	packet channeltypes.Packet,
 	_ sdk.AccAddress,
 ) ibcexported.Acknowledgement {
+	logger := im.keeper.Logger(ctx)
 	if !im.keeper.IsHostEnabled(ctx) {
+		logger.Info("host submodule disable")
 		return channeltypes.NewErrorAcknowledgement(types.ErrHostSubModuleDisabled)
 	}
 
 	txResponse, err := im.keeper.OnRecvPacket(ctx, packet)
 	ack := channeltypes.NewResultAcknowledgement(txResponse)
 	if err != nil {
+		logger.Error(fmt.Sprintf("failed to handle packet sequence %d", packet.Sequence))
```

Comment:
Couldn't we log here the error returned by `im.keeper.OnRecvPacket(ctx, packet)` above and remove the logging from `OnRecvPacket`?